### DISCUSSION TITLE
Your team is $20 overdue

### DISCUSSION THREAD

**Discussion by seanlail** [vercel/community#534] [Help] [answered]
Hi, not sure what is going on.

I created an account to test something out, I saw the "hobby" level was free.

I didn't create a team?

<img width="784" alt="image" src="https://user-images.githubusercontent.com/142589/161392779-44e21c54-221f-4a5c-b9f4-eb4abad145a8.png">

**Comment by swarnava** [comment, marked as answer]
Hi,

We need to take a closer look at your account which need to share your email address and other personal details. Could you raise a Support Case from here: https://vercel.com/support/request ?
